scripts/gen_quick_start_module.py

```diff
@@ -58,8 +59,8 @@ class OperatingSystem(Enum):
         CXX11_ABI: "Download here (cxx11 ABI):",
         RELEASE: "Download here (Release version):",
         DEBUG: "Download here (Debug version):",
-        MACOS: "Download x86 libtorch here (ROCm and CUDA are not supported):",
-        MACOS_ARM64: "Download arm64 libtorch here (ROCm and CUDA are not supported):",
+        MACOS: "Download x86 libtorch here",
+        MACOS_ARM64: "Download arm64 libtorch here",
```

Comment:
Can you please submit this change as a separate PR (and I want Greg to have a quick look at it)

Reply:
Added ROCM and CUDA not supported back